get_login.js

```diff
@@ -8,7 +8,7 @@ email = 'こちらを設定してください';
 // その時のパスワード
 password = 'こちらを設定してください';
 
-// トークンを有効とする秒数 ・「0」を指定した場合は無期限 ・デフォルトは86400秒(24時間)
+// トークンを有効とする秒数 ・「0」を指定したとき or 設定しなかったとき86400秒(24時間)
```

Comment:
ユーザ向けのものなのでもう少し説明っぽい文章にしておきたいですね。
```suggestion
// トークンを有効とする秒数 ・「0」を指定した場合や、値を指定しなかった場合はデフォルトの86400秒(24時間)になります。

```

Reply:
ecf2e10b75cd64b011a5ae7f185fbc927659ff79 で修正しました！

Reply:
👍